docs/how_tos/communicate-new-checks.rst

```diff
@@ -0,0 +1,53 @@
+How-To: Communicate New Lint Plugins or Checks
+==============================================
+
+Adding a new lint plugin
+------------------------
+
+After coding a new lint plugin, it should start off being optional, even if you intend to make it required-by-default in the future. See decisions/0001-adding-new-checks.rst for background on why.
+
+If all checks in the plugin will always be optional, simply provide instructions in the CHANGELOG.rst for how to add the plugin. For example::
+
+  * Explain the purpose of the new plugin, and potentially list its new checks.
+  * To use the ``example_plugin`` plugin, you should follow the "LOCAL CHANGES" instructions in pylintrc to add it:
+
+    .. code-block:: python
+
+        load-plugins=edx_lint.pylint.example_plugin
+
+If any checks in the plugin are meant to become required-by-default in the future:
+
+* Follow the instructions in the pylintrc file for making a "CENTRAL CHANGE".
+
+  * Add the plugin to ``load-plugins`` as detailed above in pylintrc.
+  * Ensure all new checks are added under the disable list, and not the enable list in pylintrc.
+
+* Add the new CHANGELOG.rst entry:
+
+  * Explain the purpose of the new plugin and list its new checks.
+  * List any new checks that will become required-by-default under a "Deprecated" section, noting the fact that they are disabled is deprecated, and they will be required/enabled by default, in a future major release.
```

Comment:
**Note to self:** Probably should also explain how to make "LOCAL CHANGE" in pylintrc to enable locally in advance of it becoming required.

Reply:
Not applicable, and updated README.